doc/conf.py

```diff
@@ -220,114 +197,131 @@
-#html_use_opensearch = ''
+#html_use_opensearch = ""
 
 # This is the file name suffix for HTML files (e.g., ".xhtml").
 #html_file_suffix = None
 
 # Language to be used for generating the HTML full-text search index.
 # Sphinx supports the following languages:
-#   'da', 'de', 'en', 'es', 'fi', 'fr', 'h', 'it', 'ja'
-#   'nl', 'no', 'pt', 'ro', 'r', 'sv', 'tr'
-#html_search_language = 'en'
+#   "da", "de", "en", "es", "fi", "fr", "h", "it", "ja"
+#   "nl", "no", "pt", "ro", "r", "sv", "tr"
+#html_search_language = "en"
 
 # A dictionary with options for the search language support, empty by default.
-# Now only 'ja' uses this config value
-#html_search_options = {'type': 'default'}
+# Now only "ja" uses this config value
+#html_search_options = {"type": "default"}
 
 # The name of a javascript file (relative to the configuration directory) that
 # implements a search results scorer. If empty, the default will be used.
-#html_search_scorer = 'scorer.js'
+#html_search_scorer = "scorer.js"
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'PennyLanedoc'
-
-# # -- Xanadu theme ---------------------------------------------------------
-html_theme = 'xanadu_theme'
-html_theme_path = ['.']
+htmlhelp_basename = "PennyLanedoc"
 
-# Register the theme as an extension to generate a sitemap.xml
-# extensions.append("guzzle_sphinx_theme")
+# -- Xanadu theme ---------------------------------------------------------
+html_theme = "xanadu"
 
-# xanadu theme options (see theme.conf for more information)
+# Xanadu theme options (see theme.conf for more information).
 html_theme_options = {
-
-    # Set the path to a special layout to include for the homepage
-    # "index_template": "special_index.html",
-
-    # Set the name of the project to appear in the left sidebar.
-    "project_nav_name": "PennyLane",
-
-    # Set your Disqus short name to enable comments
-    # "disqus_comments_shortname": "pennylane-1",
-
-    # Set you GA account ID to enable tracking
-    "google_analytics_account": "UA-130507810-2",
-
-    # Path to a touch icon
-    "touch_icon": "logo_new.png",
-
-    # Specify a base_url used to generate sitemap.xml links. If not
-    # specified, then no sitemap will be built.
-    # "base_url": ""
-
-    # Allow a separate homepage from the master_doc
-    # "homepage": "index",
-
-    # Allow the project link to be overriden to a custom URL.
-    # "projectlink": "http://myproject.url",
-
-    "large_toc": True,
-    # colors
-    "navigation_button": "#19b37b",
-    "navigation_button_hover": "#0e714d",
-    "toc_caption": "#19b37b",
-    "toc_hover": "#19b37b",
-    "table_header_bg": "#edf7f4",
-    "table_header_border": "#19b37b",
-    "download_button": "#19b37b",
-    # gallery options
-    # "github_repo": "PennyLaneAI/pennylane",
-    # "gallery_dirs": "tutorials",
+    "navbar_wordmark_path": "_static/pl_wordmark.png",
+    # Specifying #19b37b is more correct but does not match the other PL websites.
```

Comment:
Wait, does the navar logo colour not match the rest of the docs?

Reply:
That's right! Currently, the PennyLane navbar uses the standard Xanadu logo (which is teal-ish). We can switch the colour of the Xanadu logo to the PennyLane green once we're in a position to deploy this change across all of the PennyLane websites at once.